tests/testdata/Dockerfile-postgis

```diff
@@ -5,3 +5,16 @@ ADD auth_system/certs_keys/postgres.key /etc/ssl/private/postgres_key.key
 ADD auth_system/certs_keys/issuer_ca_cert.pem /etc/ssl/certs/issuer_ca_cert.pem
 
 RUN chmod 400 /etc/ssl/private/postgres_key.key
+
+# Compile and install PointCloud.
+# NOTE: release 1.2.0 would not build against PostgreSQL-11:
```

Comment:
The last 1.2.1 release  should compile fine though.

Reply:
Trying 1.2.1 next...